### ISSUE TITLE
--columns option problem

### ISSUE THREAD

**Issue by staszekdh** [iquasere/UPIMAPI#10]
When running like this

```
python upimapi.py 
```

and manually specify a Uniprot ID (e.g. W8T227), I get a correct result that includes the column "Taxonomic lineage IDs (SPECIES)".

However, if I use the command `--columns`

```
python upimapi.py --columns "Taxonomic lineage IDs (SPECIES)"
```

I get the following error:

```
Traceback (most recent call last):
  File "/home/users/sdunin/miniconda3/envs/jup/lib/python3.10/site-packages/pandas/core/indexes/base.py", line 3621, in get_loc
    return self._engine.get_loc(casted_key)
  File "pandas/_libs/index.pyx", line 136, in pandas._libs.index.IndexEngine.get_loc
  File "pandas/_libs/index.pyx", line 163, in pandas._libs.index.IndexEngine.get_loc
  File "pandas/_libs/hashtable_class_helper.pxi", line 5198, in pandas._libs.hashtable.PyObjectHashTable.get_item
  File "pandas/_libs/hashtable_class_helper.pxi", line 5206, in pandas._libs.hashtable.PyObjectHashTable.get_item
KeyError: 'Organism'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/users/sdunin/scr/UPIMAPI/upimapi.py", line 1406, in <module>
    upimapi()
  File "/home/users/sdunin/scr/UPIMAPI/upimapi.py", line 1386, in upimapi
    result = uniprot_information_workflow(
  File "/home/users/sdunin/scr/UPIMAPI/upimapi.py", line 591, in uniprot_information_workflow
    del uniprotinfo[k]
  File "/home/users/sdunin/miniconda3/envs/jup/lib/python3.10/site-packages/pandas/core/generic.py", line 4048, in __delitem__
    loc = self.axes[-1].get_loc(key)
  File "/home/users/sdunin/miniconda3/envs/jup/lib/python3.10/site-packages/pandas/core/indexes/base.py", line 3623, in get_loc
    raise KeyError(key) from err
KeyError: 'Organism'

```

**Comment by iquasere** [comment]
Hey there! Thank you for reporting on this issue!

UPIMAPI should be able to be run simply as `upimapi` instead of `python upimapi.py`. Nevertheless, I've checked the reported error on my end and was able to reproduce it. Next week I will launch a new version of UPIMAPI with this bug fixed. Sorry about that!

**Comment by staszekdh** [comment]
Hi, thanks for the answer and the great tool - it saved me hours of work! I was also wondering if it would be possible to introduce pip installation? Something like `pip install -r requirements.txt`.

**Comment by iquasere** [comment]
Just released UPIMAPI 1.12.3, it passed the tests and experimenting with the various taxonomic columns (both "Taxonomic lineage" and "Taxonomic lineage IDs"), it seems to be working fine. If it's still not working as it should, please don't hesitate raising another issue.

> I was also wondering if it would be possible to introduce pip installation?

You mean something like `pip install upimapi`? Or just to install its dependencies?
